core/src/main/java/hudson/util/XStream2.java

```diff
@@ -97,7 +98,18 @@
      */
     private MapperInjectionPoint mapperInjectionPoint;
 
+    /**
+     * Convinience method so we only have to change the driver in one place
+     * if we switch to something new in the future
+     *
+     * @return a new instance of the HierarchicalStreamDriver we want to use
+     */
+    public static HierarchicalStreamDriver getDefaultDriver() {
+        return new KXml2Driver();
```

Comment:
Are drivers not ThreadSafe?  Can we not reuse them?

Reply:
http://x-stream.github.io/faq.html#Scalability

Yes, i will address this in a future PR